### ISSUE TITLE
Active/focus styles on navbar should be more noticable

### ISSUE THREAD

**Issue by cpsievert** [rstudio/quillt#32]
Use maybe a combination of darker color and larger weight? Or maybe add a `border-bottom-color`?

<img width="400" alt="Screen Shot 2021-04-05 at 3 54 06 PM" src="https://user-images.githubusercontent.com/1365941/113626132-635f1a00-9627-11eb-8c55-46e4e9f7e61b.png">

**Comment by apreshill** [comment]
I agree- plan is to switch over to dev pkgdown to use the new styling: https://pkgdown.r-lib.org/dev/reference/index.html
